packages/docs/src/components/side-nav.tsx

```diff
@@ -41,9 +40,7 @@ const StyledLink = styled(Link)`
   }
 
   &[aria-current='page'] {
-    border-left: 2px solid ${colors.primary};
-    padding-left: 10px;
-    margin-left: -12px;
+    color: ${colors.primary};
```

Comment:
nit: this is a minor accessibility regression, using color alone to distinguish meaning isn't as usable as the "indicator" you had before. colors.primary has sufficient contrast against the background color but not enough to the standard text color

Reply:
fixed